app/views/components/_markdown_editor.html.erb

```diff
@@ -6,12 +6,19 @@
   govspeak_path ||= nil
   edition ||= nil
   error_items ||= []
+  paste_html_to_govspeak ||= false
 
   component_classes = %w[app-c-markdown-editor govuk-form-group]
   component_classes << "govuk-form-group--error" if error_items.any?
 
   insert_items = []
 
+  if textarea[:data]
+    textarea[:data]["paste-html-to-govspeak"] = paste_html_to_govspeak
+  else
+    textarea[:data] = { "paste-html-to-govspeak": paste_html_to_govspeak }
+  end
```

Comment:
You could make this a bit more succinct with something like:

`textarea[:data] = textarea.fetch(:data, {}).merge({ "paste-html-to-govspeak": paste_html_to_govspeak })` and avoid the conditional.

Reply:
That looks much better - now updated 👍